en/02_quicktour.md

```diff
@@ -0,0 +1,276 @@
+# 2. A Quick Tour
+
+## Sample for running SQL
+
+Since we have the sbt project set up, we are ready to run SQL using ScalikeJDBC. In this example I will use the [H2 Database](http://www.h2database.com/).
```

Comment:
Since we have the sbt project set up, let's go straight to run some SQL examples via ScalikeJDBC. Here I will use the [H2 Database](http://www.h2database.com/). としてみます。